app/javascript/src/components/Dashboard/Notes/Note/index.jsx

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line react/jsx-filename-extension
+import React from "react";
+
+import { Button, Typography } from "neetoui";
+
+import Footer from "./Footer";
+import Header from "./Header";
+
+import { formatDateAndTime, calculateCreatedAgo } from "../utils";
+
+const Note = ({ NOTES_DATA }) =>
+  NOTES_DATA.map(NOTE => (
+    <>
```

Comment:
Read: https://academy.bigbinary.com/learn-react/pages/using-unique-keys-for-list-elements

Reply:
Fixed it.